### PR TITLE
PMM-3879 Added MongoDB Profiler Agent.

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

File: api/agentpb/agent.proto [hunk 1]
```diff
@@ -15,8 +15,9 @@ enum Type {
   MYSQLD_EXPORTER = 2;
   MONGODB_EXPORTER = 3;
   QAN_MYSQL_PERFSCHEMA_AGENT = 4;
-  QAN_MYSQL_SLOWLOG_AGENT = 6;
   POSTGRES_EXPORTER = 5;
+  QAN_MYSQL_SLOWLOG_AGENT = 6;
+  QAN_MONGO_PROFILER_AGENT = 7;
```
Comment (AlekSi) [inline api/agentpb/agent.proto]: Let's call it `qan-mongodb-profiler`, `qan_mongodb_profiler`, `QAN_MONGODB_PROFILER_AGENT`, `QANMongoDBProfilerAgent`, etc for consistency with Agent and Service types.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-334/

File: api/inventorypb/agents.proto [hunk 2]
```diff
@@ -798,6 +878,20 @@ service Agents {
...
     };
   }
+  // AddQANMongoDBProfilerAgent adds MongoDB Profiler QAN Agent.
+  rpc AddQANMongoDBProfilerAgent(AddQANMongoDBProfilerAgentRequest) returns (AddQANMongoDBProfilerAgentResponse) {
+    option (google.api.http) = {
+      post: "/v1/inventory/Agents/AddQANMongoDBProfilerAgent"
+      body: "*"
+    };
+  }
+  // ChangeQANMongoDBProfilerAgent changes Mongo Profiler QAN Agent.
```
Comment (AlekSi) [inline api/inventorypb/agents.proto]:
```suggestion
  // ChangeQANMongoDBProfilerAgent changes MongoDB Profiler QAN Agent.
```

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-334/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-334/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-334/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-334/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-334/

**Pull request merged**